src/main/java/xyz/aadev/generitech/common/network/messages/power/PacketSides.java

```diff
@@ -0,0 +1,79 @@
+package xyz.aadev.generitech.common.network.messages.power;
+
+import io.netty.buffer.ByteBuf;
+import net.darkhax.tesla.api.ITeslaConsumer;
+import net.minecraft.client.network.NetHandlerPlayClient;
+import net.minecraft.inventory.IInventory;
+import net.minecraft.item.ItemStack;
+import net.minecraft.network.NetHandlerPlayServer;
+import net.minecraft.tileentity.TileEntity;
+import net.minecraft.util.EnumFacing;
+import net.minecraft.util.math.BlockPos;
+import xyz.aadev.aalib.common.inventory.InventoryOperation;
+import xyz.aadev.aalib.common.network.PacketBaseThreadSafe;
+import xyz.aadev.generitech.GeneriTech;
+import xyz.aadev.generitech.common.tileentities.TileEntityMachineBase;
+import xyz.aadev.generitech.common.tileentities.power.TileEntityPowerStorage;
+
+import javax.annotation.Nullable;
+
+public class PacketSides extends PacketBaseThreadSafe {
+
+    private int side0;
+    private int side1;
+    private int side2;
+    private int side3;
+    private int side4;
+    private int side5;
+
+
+    public PacketSides(){
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.aadev.xyz/coding_rules#rule_key=squid%3AS1186)